worlds/sc2/docs/setup_fr.md

```diff
@@ -0,0 +1,214 @@
+[*YAML specification*](https://yaml.org/spec/1.2.2/#21-collections). 
+Pour les listes, chaque *item* doit être sur sa propre ligne et doit être précédé par un trait d'union.
+
+```yaml
+excluded_items:
+  - Battlecruiser
+  - Drop-Pods (Kerrigan Tier 7)
+```
+
+Une liste vide est représentée par une paire de crochets: `[]`. 
+Il s'agit de la valeur par défaut dans le modèle de base, ce qui devrait vous aider à apprendre à utiliser cette 
+syntaxe.
+
+#### Comment est-ce que je fais pour avoir des *items* dès le départ?
+
+L'option *starting inventory* est un *map* et non une liste. 
+Ainsi, elle permet de spécifier le nombre de chaque *item* avec lequel vous allez commencer.
+Sa syntaxe consiste à indiquer le nom de l'*item*, suivi par un deux-points, puis par un espace et enfin par le nombre 
+désiré de cet *item*.
+
+```yaml
+start_inventory:
+  Micro-Filtering: 1
+  Additional Starting Vespene: 5
+```
+
+Un *map* vide est représenté par une paire d'accolades: `{}`. 
+Il s'agit de la valeur par défaut dans le modèle de base, ce qui devrait vous aider à apprendre à utiliser cette 
+syntaxe.
+
+#### Comment est-ce que je fais pour connaître le nom des *items* et des *locations* dans *StarCraft 2 Archipelago*? 
+
+La page [*datapackage*](/datapackage) d'Archipelago liste l'ensemble des *items* et des *locations* de tous les jeux 
+que le site web prend en charge actuellement, dont ceux de *StarCraft 2*.
+
+Vous trouverez aussi la liste complète des *items* de *StarCraft 2 Archipelago* à la page 
+[*Icon Repository*](https://matthewmarinets.github.io/ap_sc2_icons/). 
+Notez que cette page contient diverses informations supplémentaires sur chacun des *items*.
+Cependant, l'information présente dans cette dernière peut différer de celle du *datapackage* d'Archipelago puisqu'elle 
+est générée, habituellement, à partir des développements récents de *StarCraft 2 Archipelago* qui n'ont peut-être pas 
+encore été inclus dans le site web d'Archipelago.
+
+## Comment est-ce que je peux joindre un *MultiWorld*?
+
+1. Exécuter `ArchipelagoStarcraft2Client.exe`.
+   - Uniquement pour cette étape, les utilisateurs de macOS devraient plutôt suivre les instructions à la page 
+["Exécuter sous macOS"](#exécuter-sous-macos).
+2. Entrer la commande `/connect [server ip]`.
+   - Si le *MultiWorld* est hébergé via un siteweb, l'IP du server devrait être indiqué dans le haut de la page de 
+votre *room*.
+3. Inscrivez le nom de joueur spécifié dans votre *yaml* lorsque vous y êtes invité.
+4. Si le serveur a un mot de passe, l'inscrire lorsque vous y êtes invité.
+5. Une fois connecté, aller sur l'onglet *StarCraft 2 Launcher* dans le client. Dans cet onglet, vous devriez trouver 
+toutes les missions de votre monde. Les missions qui ne sont pas disponibles présentement auront leur texte dans une 
+tonte de gris. Vous n'avez qu'à cliquer une des missions qui est disponible pour la commencer!
```

Comment:
Typo (or disagreement between different regions ?) : "tonte de gris" should be "nuance de gris"

Reply:
Applied as suggested!